### DISCUSSION TITLE
HmIP-HAP (Repeater Configuration) - Update from 3.0.16 to 3.0.18 is not working.

### DISCUSSION THREAD

**Discussion by bandimuc** [OpenCCU/OpenCCU#2706] [🇺🇸 Q&A] [answered]
### Describe the issue you are experiencing

Upgrade fails after starting the process in CCU3 console. After some time the update button is showing up again.

### Describe the behavior you expected

Upgrade to 3.0.18 will be successful.

### Steps to reproduce the issue

1. Upload 3.0.18 file to CCU
2. Press Update button
3. Check progress in Firmware overview portal
...

### What is the version this bug report is based on?

3.75.6.20240316

### Which base platform are you running?

ova (Open Virtual Infrastructure, AMD64/x86_64)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
no errors found in log file
```

**Comment by Baxxy13** [comment, marked as answer]
Ich habe erst gestern einen original eQ-3 HAP an einer aktuellen RaspberryMatic (Pi3B+) von 3.0.16 auf 3.0.18 geupdated und konnte dabei keine Probleme beobachten.
Die ganze Prozedur hat etwa 2-3min gedauert.

Lösche doch den HAP mal komplett aus der Zentrale und lerne ihn neu an, vielleicht passt dann alles.

**Comment by bandimuc** [comment]
Hi Baxxy13, dank für den Hinweis. Nachdem ich den HAP aus der CCU gelöscht, den HAP 2x auf Werkseinstellung zurückgesetzt habe, hat es sich nach dem Anlernen auf 3.0.18 updaten lassen :)

Danke
